.github/workflows/check_smoke_tests.yml

```diff
@@ -65,12 +65,14 @@ jobs:
 
           API_ROOT="/api"
           BASE_URL="http://localhost:8080${API_ROOT}"
+          INITIAL_ALGORITHMS="src/recordlinker/assets/testing_algorithms.json"
```

Comment:
I think you'll want to delete `src/recordlinker/` from this value.  Reading in the value will [start from the "recordlinker" directory](https://github.com/CDCgov/RecordLinker/blob/main/src/recordlinker/utils/path.py#L23).